index.js

```diff
@@ -1,56 +1,78 @@
 const inputAuthor = document.querySelector('.author');
 const alertMessage = document.querySelector('#alert-message');
 let theBooks = [];
 
-// const storage = localStorage;
+class BookObject {
+  constructor(title, author, id) {
+    this.title = title;
+    this.author = author;
+    this.id = id;
+  }
 
-function removeBook() {
-  theBooks = theBooks.filter((book) => +book.id !== +this.parentNode.childNodes[2].innerHTML);
-  this.parentNode.remove();
+  add() {
+    function creaateAndAppend(title, author, id) {
+      const bookContainer = document.createElement('div');
+      const infoContainer = document.createElement('div');
+      const titleBook = document.createElement('h5');
+      const authorBook = document.createElement('h5');
+      const idBook = document.createElement('h5');
+      const removeButton = document.createElement('button');
 
-  theBooks.forEach((book, i) => {
-    bookList.children[i].children[2].innerHTML = i;
-    book.id = i;
-  });
-  localStorage.setItem('booksArray', JSON.stringify(theBooks));
-}
+      bookContainer.classList.add('book-container');
+      infoContainer.classList.add('book-info');
+      titleBook.classList.add('book-title');
+      authorBook.classList.add('book-author');
+      removeButton.classList.add('remove-btn');
 
-function creaateAndAppend(title, author, id) {
-  const bookContainer = document.createElement('div');
-  const titleBook = document.createElement('h5');
-  const authorBook = document.createElement('h5');
-  const idBook = document.createElement('h5');
-  const removeButton = document.createElement('button');
-  const breakLine = document.createElement('hr');
+      titleBook.innerHTML = `${title}&nbsp;`;
+      authorBook.innerHTML = `by ${author}&nbsp;`;
+      idBook.innerHTML = id;
+      removeButton.innerHTML = 'Remove';
 
-  titleBook.innerHTML = title;
-  authorBook.innerHTML = author;
-  idBook.innerHTML = id;
-  removeButton.innerHTML = 'Remove';
+      idBook.style.display = 'none';
 
-  // idBook.style.display = 'none';
+      infoContainer.append(titleBook, authorBook, idBook);
+      bookContainer.append(infoContainer, removeButton);
+      bookList.append(bookContainer);
+
+      BookObject.remove(removeButton);
+    }
+    creaateAndAppend(this.title, this.author, this.id);
+  }
 
-  bookContainer.append(titleBook, authorBook, idBook, removeButton, breakLine);
-  bookList.append(bookContainer);
+  static remove(element) {
+    function removeBook() {
+      theBooks = theBooks.filter((book) => +book.id
+      !== +this.parentNode.children[0].children[2].innerHTML);
+      this.parentNode.remove();
 
-  removeButton.addEventListener('click', removeBook);
+      theBooks.forEach((book, i) => {
+        bookList.children[i].children[0].children[2].innerHTML = i;
+        book.id = i;
+      });
+      localStorage.setItem('booksArray', JSON.stringify(theBooks));
+    }
+
+    element.addEventListener('click', removeBook);
+  }
 }
 
 function addBooks() {
```

Comment:
- [ ] Kindly move this **`addBooks()`** method into the **BookObject** class.

Reply:
`addBooks()` equals to function that is used to add books when pressing Add button
I'm using the class and class method inside `addBooks()`